autogtp/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-add_executable(autogtp
+add_executable(leelaz-autogtp
```

Comment:
Let's keep the autogtp name. Low risk of clashes and it might be what people expect by now.